Jenkinsfile

```diff
@@ -799,8 +799,9 @@ pipeline {
                         }
                         */
                         always {
+                            sh '''mv test_results/*.xml run_test.sh/'''
+			    junit 'test_results/*.xml'
```

Comment:
Looks like you have a whitespace issue here.

Reply:
Will do... Presently, fixed on my workspace. I thought running checkpatch should find this issue. Am I wrong?